oaknational/Oak-Web-Application
# [CUR-888] Curriculum overview explainer

## Description
This PR adds the basic structure for "curriculum explainers" but is missing the styling. It is disabled by default behind the `ENABLE_CYCLE_2` constant.

- Added global constant to disable/enable new cycle 2 features 
- Added basic structure of explainer to docx
- Added basic structure of explainer to overview tab 

## Issue(s)

Fixes `CUR-888`

## How to test

1. Go to curriculum visualiser
2. Check for regressions in overview tab and downloads. Original explainers should still be present in PR branch.
